docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+
+  backend:
+    image: python:3.5
+    command: python3 /app/refactor_me.py
+    environment:
+      - PYTHONUNBUFFERED=1
+    volumes:
+      - ./:/app
+
+  flake:
+    image: flake:anton
+    command: bash -c "cd /home/docker && gulp"
```

Comment:
is there a way not to use a tool from JavaScript world?

Reply:
there is a problem with that, we can talk about it on a call :) I'll explane you my setup.